jobs/future-effective-filings/file_future_effective.py

```diff
@@ -134,9 +136,10 @@ async def run(loop, application: Flask = None):
             application.logger.error(err)
 
 if __name__ == '__main__':
+    application = create_app()
     try:
         event_loop = asyncio.get_event_loop()
-        event_loop.run_until_complete(run(event_loop))
+        event_loop.run_until_complete(run(event_loop, application))
     except Exception as err:  # pylint: disable=broad-except; Catching all errors from the frameworks
-        print('problem in running the service: %s', err, stack_info=True, exc_info=True)
+        application.logger.error(err)
```

Comment:
If this error requires human intervention to fix, it should get put on to Sentry

Reply:
I thought this did that, this has now been changed...hopefully better now